runtime/doc/news.txt

```diff
@@ -219,6 +219,11 @@ EVENTS
   for completion being done.
 • |vim.on_key()| callbacks can consume the key by returning an empty string.
 
+HIGHLIGHTS
+
+• |TermCursorNC| is removed and no longer supported. Unfocused terminals no
+  longer have a cursor.
```

Comment:
is this intended to be in `news-breaking` ?

Reply:
Yes, thanks